ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -52,6 +70,8 @@ public class ThreeDSecureRequest implements Parcelable {
     private Boolean cardAddChallengeRequested;
     private ThreeDSecureV2UiCustomization v2UiCustomization;
     private ThreeDSecureV1UiCustomization v1UiCustomization;
+    private @ThreeDSecureUiType int uiType;
+    private ArrayList<ThreeDSecureRenderType> renderType;
```

Comment:
```suggestion
    private List<Integer> renderType;
```
We may have to do it like this. Stack Overflow [breaks it down some](https://stackoverflow.com/questions/34618089/android-support-annotations-how-to-use-intdef-stringdef-typedef-annotations), but the `@IntDef` stuff is limited.

Reply:
Gotcha, updated here: https://github.com/braintree/braintree_android/pull/738/commits/b1313fc2f152519e3fea12c0de3b51e48e1364f7